appium/options/ios/xcuitest/other/other_options.py

```diff
@@ -0,0 +1,155 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import timedelta
+
+from appium.options.transformers import OptionsDescriptor
+from appium.options.common.supports_capabilities import SupportsCapabilities
+
+
+class UseJsonSourceOption(SupportsCapabilities):
+    USE_JSON_SOURCE = 'useJSONSource'
+    use_json_source = OptionsDescriptor('USE_JSON_SOURCE')
```

Comment:
are you sure the correct value is passed there? We need the option name in capabilities, not the class property name.

Reply:
> are you sure the correct value is passed there? We need the option name in capabilities, not the class property name.

Yes, it is correct. I just verified with original `getter` and `setter` method.

Reply:
@mykola-mokhnach I have incorporated all the review comments. Could you please re-check once let me know your comments.